### PR TITLE
test(integer): add unit and doc test for bitnot operator

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

File: tfhe/src/integer/server_key/radix_parallel/tests.rs [hunk 1]
```diff
@@ -716,6 +717,39 @@ where
     }
 }
 
+fn integer_default_bitnot<P>(param: P)
+where
+    P: Into<PBSParameters>,
+{
+    let (cks, sks) = KEY_CACHE.get_from_params(param);
+    let cks = RadixClientKey::from((cks, NB_CTXT));
```
Comment (IceTDrinker) [inline tfhe/src/integer/server_key/radix_parallel/tests.rs:725]: You need to set deterministic execution, check other default tests

File: tfhe/src/integer/server_key/radix_parallel/tests.rs [hunk 2]
```diff
@@ -716,6 +717,39 @@ where
...
+
+    for _ in 0..NB_TEST {
+        // Define the cleartexts
+        let clear = rng.gen::<u64>() % modulus;
+
+        // Encrypt the integers
+        let ctxt = cks.encrypt(clear);
+
+        // Negates the ctxt
+        let ct_tmp = sks.bitnot_parallelized(&ctxt);
```
Comment (IceTDrinker) [inline tfhe/src/integer/server_key/radix_parallel/tests.rs:741]: Missing determinism check for default ops, check how other default tests are written

File: tfhe/src/integer/server_key/radix_parallel/bitwise_op.rs [hunk 3]
```diff
@@ -486,6 +486,41 @@ impl ServerKey {
...
+    ///
+    /// let msg = 14;
+    ///
+    /// let ct = cks.encrypt(msg);
+    ///
+    /// let ct_res = sks.bitnot_parallelized(&ct);
+    ///
+    /// // Decrypt:
+    /// let dec_result: u64 = cks.decrypt(&ct_res);
+    /// assert_eq!(dec_result, (!msg) % (1 << 8));
```
Comment (IceTDrinker) [inline tfhe/src/integer/server_key/radix_parallel/bitwise_op.rs:522]: let msg_modulus = PARAM_MESSAGE_2_CARRY_2_KS_PBS.message_modulus.0 as u64 ?

Comment (IceTDrinker) [reply tfhe/src/integer/server_key/radix_parallel/bitwise_op.rs:522]: I sure other tests do something similar

Comment (IceTDrinker) [reply tfhe/src/integer/server_key/radix_parallel/bitwise_op.rs:522]: ah no the full modulus needs to be num_blocks * PARAM_MESSAGE_2_CARRY_2_KS_PBS.message_modulus.0

Comment (soonum) [reply tfhe/src/integer/server_key/radix_parallel/bitwise_op.rs:522]: We agreed with @jborfila during setting up this doc test to stay on `1 << 8` value to keep the doc test simple.

Comment (IceTDrinker) [reply tfhe/src/integer/server_key/radix_parallel/bitwise_op.rs:522]: ok

**Review by IceTDrinker — APPROVED** [review]
Looks good thanks !

**Pull request merged**